corehq/apps/reports/filters/api.py

```diff
@@ -158,6 +158,18 @@ def get_locations_query(self, query):
                 .filter_path_by_user_input(self.domain, query)
                 .accessible_to_user(self.request.domain, self.request.couch_user))
 
+    def get_users(self, query, start, size):
+        users = (self.user_es_query(query)
+                 .fields(['_id', 'username', 'first_name', 'last_name', 'doc_type'])
+                 .start(start)
+                 .size(size)
+                 .sort("username.exact"))
+        if not self.request.can_access_all_locations:
+            user_location_id = self.request.couch_user.get_location_id(self.domain)
+            all_location_ids = SQLLocation.location_and_descendants_ids([user_location_id])
+            users = users.location(all_location_ids)
```

Comment:
Again there is a subclass to `EmwfOptionsView`, `LocationGroupFilter` which relies on `get_users` in the parent class. Thats why added these changes in the other two classes.